app/src/main/java/com/android/streetworkapp/ui/event/AddEvent.kt

```diff
@@ -104,6 +107,7 @@ fun AddEventScreen(
           modifier = Modifier.fillMaxWidth(),
           verticalArrangement = Arrangement.spacedBy(18.dp),
           horizontalAlignment = Alignment.CenterHorizontally) {
+            Spacer(modifier = Modifier.size(24.dp))
```

Comment:
I recommend that you add in the PR description that you've modified some UI elements, so that the description is up to date.

Reply:
It is done now, thank you!